src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
+const ANIMAL_DAMAGE = 50;
+const ANIMAL_BASE_HP = 100;
+const MINIMUM_HP = 0;
```

Comment:
Change "MINIMUM_HP" to "ANIMAL_MINIMUM_HP" to make the constant names consistent.